docs/docs/guides/how-to-compose-your-own-testnet.md

```diff
@@ -130,7 +124,7 @@ And that is it! In these short few lines, you now have an environment definition
 ### 4. Run your new testnet!
 Finally, time to give it a spin! Go back to your terminal & from within the `my-testnet` directory, run:
```

Comment:
Writing style thing - using "&" instead of "and" in long-form text is very nonstandard (but don't just take my word for it - [source](https://onlinewritingtraining.com.au/ampersands/#:~:text=Answer%3A%20You%20can%20use%20ampersands,part%20of%20an%20organisation's%20branding.&text=Use%20and%2C%20not%20ampersands%20in,It%20is%20more%20professional.) and [source](https://grammarist.com/usage/ampersand/#:~:text=Is%20It%20Unprofessional%20to%20Use%20an%20Ampersand%3F)). You're not even going to see it in docs for startups, for example. Generally, you only use ampersand in places where space is limited - titles, buttons, etc.